docs/apache-airflow/configurations-ref.rst

```diff
@@ -32,6 +32,13 @@ generated using the secret key has a short expiry time though - make sure that t
 that you run airflow components on is synchronized (for example using ntpd) otherwise you might get
 "forbidden" errors when the logs are accessed.
 
+Some of the providers have their own configuration options, you will find details of their configuration
+in the provider's documentation. The pre-installed providers that you likely want to configure are:
```

Comment:
```suggestion
in the provider's documentation. The pre-installed providers that you may want to configure are:
```

I think we should soften this, 2 of the 3 common executors wouldn't need to touch this.

Reply:
Yep.